greedy_methods/fractional_knapsack.py

```diff
@@ -1,9 +1,22 @@
 from bisect import bisect
 from itertools import accumulate
 
+# Reference Link: https://en.wikipedia.org/wiki/Continuous_knapsack_problem
 
-def frac_knapsack(vl, wt, w, n):
+
+def frac_knapsack(vl: list, wt: list, w: int, n: int) -> float:
```

Comment:
Let's use self-documenting variable names...
```suggestion
def frac_knapsack(values: list, weights: list, capacity: int, max_items: int) -> float:
```